run.sh

```diff
@@ -424,9 +418,57 @@ function add_scheduler(){
     echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Added Job Scheduler for run Cloud Function."
 }
 
+ 
+# Init script with proper message and display active user account
+
+function gcloud_init_confs(){
+    user_active_account="$(gcloud auth list --filter=status:ACTIVE --format="value(account)")"
+    echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Your active account [${user_active_account}]"
+    echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Choose Project ID"
+    _choose_and_set_project_id
+	echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Project ID was updated."
+}
+
+
+# Choose and set project id
+# Error:
+#   Exit Code 1
+function _choose_and_set_project_id(){
+    array_projects=()
+    project_id=""
+    count=0
+    for project in  $(gcloud projects list --format="value(projectId)")
+    do
+        count=$((count + 1))
+        echo "[$count]:  $project"
+        array_projects+=("$project")
+    done
+    read -n 2 -p "Please fill in number of the project where you would like the integration to be deployed in: " mainmenuinput
```

Comment:
Not sure that I understand this instruction. What does number of project mean? Is that the Id?

Reply:
In that cause it will be display list of the projects
[1] project_id_1
[2]project_id_2
[3]project_id_3

and user need to fill a number 3 to set up as project_id_3 as project where  need to be run integration

Reply:
Can you consult with @nico-shishkin on how to phrase this line?

Reply:
Hi guys, I think "Please provide the ID number (e.g., 3 for project_id_3) of the project where you would like to deploy the integration"

Reply:
@mirii1994 Please specify the project  for integration deployment by referring to the project's numerical position in the list.
This is our message after an internal discussion

Reply:
Great, thanks